app/src/hooks/useSessionStorage.ts

```diff
@@ -0,0 +1,56 @@
+import { useState } from 'react'
```

Comment:
Notes for reviewers: The main differences between `useLocalStorage.tsx` and `useSessionStorage.ts` is that one uses the browser's local storage (non-ideal for our purposes because the data persists for too long and is not friendly for public shared computers) and the other uses the browser's session storage (cleared when the browser tab/window is closed).